Sources/Auth/AuthClient.swift

```diff
@@ -34,7 +34,7 @@ public actor AuthClient {
       url: URL,
       headers: [String: String] = [:],
       flowType: AuthFlowType = Configuration.defaultFlowType,
-      localStorage: AuthLocalStorage = Configuration.defaultLocalStorage,
+      localStorage: AuthLocalStorage,
```

Comment:
We can no longer guarantee a default since there is no good way to provide a standard implementation in Linux (there is no similar Keychain/WinCred API)